### PR TITLE
Remove 'copy application' functionality on exporter side

### PR DESCRIPTION
**Pull request by hnryjmes**
This change removes the 'copy application' functionality from exporter.

Only the frontend changes have been made at this time.

Before / After:

**Remove button on 'application' page:**
Screenshot 2023-01-30 at 14 48 52
Screenshot 2023-01-30 at 14 48 59

**Remove ‘actions’ column in submitted tab:**
Screenshot 2023-01-30 at 14 48 23
Screenshot 2023-01-30 at 14 48 30

**Remove ‘copy’ hyperlink in drafts tab:**
Screenshot 2023-01-30 at 14 31 51
Screenshot 2023-01-30 at 14 31 58

### CODE REVIEW

**Review by depsiatwal — COMMENTED** [review]
changes LGTM 
I will approve once e2e tests are running as I suspect we could have some e2e test may fail

**Comment by hnryjmes** [comment]
> changes LGTM I will approve once e2e tests are running as I suspect we could have some e2e test may fail

no worries, should be good now

**Review by depsiatwal — APPROVED** [review]
lgtm

File: ui_tests/exporter/features/copy_application.feature [hunk 1]
```diff
@@ -1,19 +0,0 @@
-@copy_application @all
-Feature: I want to add a note to an application and view notes
-  As a logged in exporter
-  I want to add a note to an application and view existing notes
-  So that I can record my findings and comments and others users can see these
-
-  @skip @legacy
```
Comment (saruniitr) [inline ui_tests/exporter/features/copy_application.feature:7]: this test is already skipped so no need to delete

**Pull request merged**